cli/src/cluster_query.rs

```diff
@@ -369,14 +381,18 @@ pub fn parse_catchup(
     matches: &ArgMatches<'_>,
     wallet_manager: &mut Option<Arc<RemoteWalletManager>>,
 ) -> Result<CliCommandInfo, CliError> {
-    let node_pubkey = pubkey_of_signer(matches, "node_pubkey", wallet_manager)?.unwrap();
+    let node_pubkey = pubkey_of_signer(matches, "node_pubkey", wallet_manager)?;
     let node_json_rpc_url = value_t!(matches, "node_json_rpc_url", String).ok();
+    let guess = matches.is_present("our_localhost");
```

Comment:
`node_pubkey.is_none() && !guess` is an invalid combination, no?

Reply:
hehe, good catch; you're right. the error condition is handled there: https://github.com/solana-labs/solana/pull/14313/files#r550108108

Reply:
I'll move here.